Doc/howto/clinic.rst

```diff
@@ -1389,18 +1406,18 @@ Writing a custom return converter is much like writing
 a custom converter.  Except it's somewhat simpler, because return
 converters are themselves much simpler.
 
-Return converters must subclass ``CReturnConverter``.
+Return converters must subclass :py:class:`!CReturnConverter`.
 There are no examples yet of custom return converters,
 because they are not widely used yet.  If you wish to
-write your own return converter, please read ``Tools/clinic/clinic.py``,
-specifically the implementation of ``CReturnConverter`` and
+write your own return converter, please read :source:`Tools/clinic/clinic.py`,
+specifically the implementation of :py:class:`!CReturnConverter` and
 all its subclasses.
 
 
 How to convert ``METH_O`` and ``METH_NOARGS`` functions
 -------------------------------------------------------
 
-To convert a function using ``METH_O``, make sure the function's
+To convert a function using :data:`METH_O`, make sure the function's
```

Comment:
Use `:c:macro:` for `METH_*` constants.

Reply:
Thanks (and thanks for fixing these recently!). Resolved with 1a30f3b661622d07ff29b0aafc8dda4546c5a999